docs/SETUP-QUICKSTART.md

```diff
@@ -133,10 +135,10 @@ heroku.app: dsr-min-cgaucho
 heroku.url: https://dsr-min-cgaucho.herokuapp.com
 auth0.tenant: ucsb-cs156-cgaucho
 auth0.domain: ucsb-cs156-cgaucho.us.auth0.com
-auth0.clientid: 6KoPsWMM2A27PjAejHHWTXApra8CVQ6C
-google.clientId:
-google.clientSecret: 
+auth0.clientid: 6KoPsWMM2A27PjAejHHWTXApra8CVQ6C 
```

Comment:
Very minor, but shouldn't the `i` in `id` be capitalized?

Reply:
Yes it should.  I'll fix it today in class as a demo.